_posts/2019/2019-10-29-Snap-GE-statement.md

```diff
@@ -0,0 +1,27 @@
+---
+date:         2019-10-29
+category:     blog
+tags:
+  - Elections
+layout:       post
+title:        "General Election Statement"
+image:        /assets/img/articles/passport.jpg
+author:       Harley Faggetter
+---
+
+Dear Pirates,
+
+Pirate Party UK believes that leaving the EU without a deal goes against our Party principles. In particular, we believe it will result in the likely degradation of civil liberties, particularly through past Tory campaigns to water down or replace the Human Rights Act, and also digital rights and freedoms given through the EU, such as the protections of the GDPR and access to the Horizon 2020 Open Access programmes.
```

Comment:
```suggestion
Pirate Party UK believes that leaving the EU without a deal goes against our Party principles. In particular, we believe it would result in the likely degradation of civil liberties, particularly because of previous Tory campaigns to water down or replace the Human Rights Act and to dilute digital rights and freedoms granted through the EU, such as the protections of the GDPR. We also believe that we would lose important access to the scientific community in Europe through programmes such as Horizon 2020 and Erasmus.
```